### PR TITLE
fix: add overflow handling to label chooser in create sample view to prevent overflow

### PR DESCRIPTION
**Pull request by elanjm12**
Overflow handling added to label chooser in create sample view

Screenshot from 2022-05-13 10-30-34
.

### CODE REVIEW

**Review by igboyes — CHANGES_REQUESTED** [review]
Can you tweak this so the filter input component stays visible at the top and only the label list scrolls?

**Comment by elanjm12** [comment]
filter input component remains visible while scrolling 

Screenshot from 2022-05-15 15-59-49

File: src/js/samples/components/Sidebar/Selector.js [hunk 1]
```diff
@@ -69,10 +74,12 @@ export const SampleSidebarSelector = ({
                             autoFocus
                         />
                     </SampleSidebarSelectorInputContainer>
-                    {sampleItemComponents}
-                    <StyledLinkButton>
-                        <Link to={manageLink}> Manage</Link>
-                    </StyledLinkButton>
+                    <StyledDiv>
```
Comment (igboyes) [inline src/js/samples/components/Sidebar/Selector.js]: `StyledLinkButton` is incorrectly named, so don't follow that as an example.

The Styled Components convention is to use the `Styled` prefix only in the following situation:
```js
const StyledBar = styled.div`...`;

const Bar = props => (
  <StyledBar>
    ...
  </StyledBar>
);
```

https://styled-components.com/docs/basics#how-do-styled-components-work-within-a-component

---

Can you rename `StyledLinkButton` to `SampleSidebarSelectorButton`?

**Review by igboyes — CHANGES_REQUESTED** [review]
Please also change commit message to something a user will understand (eg. _prevent excessive label selector menu heights_).

**Review by igboyes — APPROVED** [review]

**Pull request merged**